lib/lambda_ethereum_consensus/fork_choice/fork_choice.ex

```diff
@@ -70,7 +72,7 @@ defmodule LambdaEthereumConsensus.ForkChoice do
   @spec init({BeaconState.t(), SignedBeaconBlock.t(), Types.uint64()}) ::
           {:ok, Store.t()} | {:stop, any}
   def init({anchor_state = %BeaconState{}, signed_anchor_block = %SignedBeaconBlock{}, time}) do
-    case Helpers.get_forkchoice_store(anchor_state, signed_anchor_block, true) do
+    case Helpers.get_forkchoice_store(anchor_state, signed_anchor_block, false) do
```

Comment:
I've had to turn off db because otherwise the `get_head` function fails.

Reply:
I'll take a look